models/index.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+
+const basename = path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require('../config/db.json')[env];
+
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
```

Comment:
This should be done transparently and all based on a single config file.  Let's add a ticket for that and follow up discussion